src/gba/vfame.h

```diff
@@ -0,0 +1,34 @@
+/* Copyright (c) 2016 taizou
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Support for copy protected unlicensed games from the company Vast Fame
+
+#ifndef GBA_VFAME_H
+#define GBA_VFAME_H
+
+#include "util/common.h"
+
+enum GBAVFameCartType {
+	VFAME_NO = 0,
+	VFAME_STANDARD = 1,
+	VFAME_GEORGE = 2
+};
+
+struct GBAVFameCart {
+	enum GBAVFameCartType cartType;
+	int sramMode;
+	int romMode;
+	int8_t writeSequence[5];
+	char acceptingModeChange;
+};
```

Comment:
`bool`

Also at some point, these might need to go into save states, especially for TAS reasons.